content/docs/esc/reference.md

```diff
@@ -54,6 +54,12 @@ values:
     # Array elements are "app.items[0]" and "app.items[1]"
     items: [ "config-a", "config-b" ]
 
+    # Path is "app.multiline"
+    # If the value needs to be a multiline value, use YAML pipe.
```

Comment:
You might say "use a YAML block" and provide a link to https://yaml-multiline.info/